<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_bt_dish_kitchenprinter_printerset_port</class>
 <widget class="QWidget" name="w_bt_dish_kitchenprinter_printerset_port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_usb_vp">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定打印机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_port_tty">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_port_usb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_baud">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_flowcontrol">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>流控</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_baud">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="lds_ComboBox" name="comboBox_flowcontrol">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="lds_ComboBox" name="comboBox_porttype">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>端口类型</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
